examples/with_winit/src/lib.rs

```diff
@@ -142,6 +147,23 @@ fn run(
                             Some(VirtualKeyCode::Space) => {
                                 transform = Affine::IDENTITY;
                             }
+                            Some(VirtualKeyCode::S) => {
+                                stats_shown = !stats_shown;
+                            }
+                            Some(VirtualKeyCode::C) => {
+                                stats.clear_min_and_max();
+                            }
+                            Some(VirtualKeyCode::V) => {
+                                vsync_on = !vsync_on;
+                                render_cx.set_present_mode(
+                                    &mut render_state.surface,
+                                    if vsync_on {
+                                        wgpu::PresentMode::Fifo
+                                    } else {
+                                        wgpu::PresentMode::Immediate
```

Comment:
```suggestion
                                        wgpu::PresentMode::AutoNoVsync
```
Should be supported on all platforms. There's also a question of doing `PresentMode::AutoVsync` for vsync. I don't have a machine which it would make a difference on; apparently it's only Amd Vulkan.

The alternative might also resolve #266

Reply:
Done, I change the code to always use the `Auto*` modes. On web this likely means that "vsync off" will be incorrect so maybe the stats UI shouldn't just blindly display "VSync: off", but it's probably not a huge deal.